analysis/repeat_events/process_repeat_events.R

```diff
@@ -101,13 +97,19 @@ for (cohort in c("prevax", "unvax", "vax")) {
 
   # Reshape stage1 data ----------------------------------------------------------
   stage1_cohort_long <- stage1_cohort %>%
+    select(-exposure_date) %>%
     pivot_longer(
       cols = contains("date"),
       names_to = "date_label",
       values_to = "date",
       values_drop_na = TRUE
     )
   
+  # Keep exposure dates for later -----------------------------------------------
+  data_exposure_dates <- stage1_cohort %>%
+    select(patient_id, exposure_date) %>%
+    filter(!is.na(exposure_date))
+  
```

Comment:
Although this requires a join later, I think this is the best way to add a column of exposure dates.